src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/FormTheme/combination_listing.html.twig

```diff
@@ -29,14 +29,50 @@
     {# Special case to keep both inputs in the same cell #}
     {% for child in form.children %}
       <td>
-        {# When value is null we are displaying the prototype #}
-        {% if value is null %}
-          {% set widgetAttr = {value: placeholder_data[child.vars.name]} %}
-        {% else %}
-          {% set widgetAttr = {} %}
+        {# action column is rendered separately bellow #}
+        {% if child.vars.name != 'actions' %}
+          {# When value is null we are displaying the prototype #}
+          {% if value is null %}
+            {% set widgetAttr = {value: placeholder_data[child.vars.name]} %}
+          {% else %}
+            {% set widgetAttr = {} %}
+          {% endif %}
+          {{ form_widget(child, widgetAttr) }}
+          {{ form_errors(child) }}
+        {% elseif child.children is not empty %}
```

Comment:
I think there's a better alternative to handle this specific display:
- `CombinationListType` has a dedicated theme `@PrestaShop/Admin/Sell/Catalog/Product/FormTheme/combination_listing.html.twig` in this theme you could override the actions ButtonCollectionType field to match your need, ideally with a dedicated `block_prefix` for this field so that it's the only one impacted (in case another ButtonCollectionType is added in the form item someday or by a module)
- an even more generic approach would be to extend the possibilities of `ButtonCollectionType` directly in the ui_kit_base theme, by adding for example a `dropdow_buttons` option (false by default) that would handle this dropdown display, it would make this component more reusable ([theme widget here](https://github.com/PrestaShop/PrestaShop/blob/3ca279fc8223d423eca55eb4ba3ddeeb50e6d0ff/src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig#L1486)) I think this approach has more potential and is cleaner

Reply:
Second thought about the `ButtonCollectionType` approach that your current implementation inspired me, instead of a boolean `dropdown_buttons` option, it ould maybe be more interesting to have an int like `max_inline_buttons` that would replace your current `skippedActions` internal variable, so that we can display one two or maybe even 0 buttons before the dropdown (default would be false meaning they are all inline and now in the dropdown) What do you think?